lib/x11test.pm

```diff
@@ -556,56 +554,58 @@ sub start_clean_firefox {
 
     # Help
     send_key "alt-h";
-    sleep 1;
+    assert_screen 'firefox-help-menu';
     send_key "a";
     assert_screen('firefox-help', 30);
     send_key "esc";
 
-    # store .mozilla configuration as default to avoid popup checks in following tests
-    $self->restart_firefox('cp -rp .mozilla .mozilla_first_run');
+    # restart firefox to trigger default browser pop-up and store .mozilla configuration as default without pop-ups
+    $self->restart_firefox('sync && cp -rp .mozilla .mozilla_first_run', 'opensuse.org');
 }
 
 sub start_firefox_with_profile {
-    my ($self) = @_;
+    my ($self, $url) = @_;
+    $url ||= '/home';
```

Comment:
`||=` vs `//=` - does not matter here, just for info
`||=` - if empty string or the number 0 are valid parameters then they are replaced by default value